_extension/src/util.ts

```diff
@@ -0,0 +1,91 @@
+import * as path from "path";
+import * as vscode from "vscode";
+import packageJson from "../package.json";
+
+const version = packageJson.version;
+
+export const jsTsLanguageModes = [
+    "typescript",
+    "typescriptreact",
+    "javascript",
+    "javascriptreact",
+];
+
+export const builtinTSExtensionId = "vscode.typescript-language-features";
+
+export interface ExeInfo {
+    path: string;
+    version: string;
+}
+
+export function getBuiltinExePath(context: vscode.ExtensionContext): string {
+    return context.asAbsolutePath(path.join("./lib", `tsgo${process.platform === "win32" ? ".exe" : ""}`));
```

Comment:
I'd like to be able to move this to `node_modules` so we can just use resolve, but vsce is making it very hard to do that due to https://github.com/microsoft/vscode-vsce/issues/970, so I think we'll have to keep this for now. Thankfully it's in the VSIX so is changeable, unlike the other resolution algorithm.